cmd/skipper/main.go

```diff
@@ -591,6 +591,10 @@ func main() {
 		Append:  appendFilters.Get(),
 	}
 
+	if apiUsageMonitoringDefaultClientTrackingPattern != defaultApiUsageMonitoringDefaultClientTrackingPattern {
+		log.Warn(`"api-usage-monitoring-default-client-tracking-pattern" parameter is deprecated`)
```

Comment:
this warning will not warn library users, so please also add a warning for library users.

Reply:
done + rebased